packages/common/src/scopeSupportFacets/scopeSupportFacetInfos.ts

```diff
@@ -225,6 +225,23 @@ export const scopeSupportFacetInfos: Record<
     scopeType: "string",
   },
 
+  "textFragment.comment.line": {
+    description: "A line comment",
+    scopeType: "comment",
+  },
+  "textFragment.comment.block": {
+    description: "A block comment",
+    scopeType: "comment",
+  },
+  "textFragment.string.singleLine": {
+    description: "A single-line string",
+    scopeType: "string",
+  },
+  "textFragment.string.multiLine": {
+    description: "A multi-line string",
+    scopeType: "string",
+  },
```

Comment:
These scope types and descriptions should be updated, no?

Reply:
Well the description is still apt, but I guess i can eaborate a bit

Reply:
Yeah would be good to indicate that they're text fragments. And the scope types are wrong, no?

Reply:
You're definitely right. Done